### PR TITLE
Add establishment search results page

### PR DESCRIPTION
**Pull request by wheelsandcogs**
Allows the user to search for establishments and displays the results in our standard datatable format.

### CODE REVIEW

**Comment by lennym** [comment]
This supercedes the establishment list that is currently mounted on `/establishments` and so that should be removed.

**Comment by wheelsandcogs** [comment]
Are we removing the "View all" links from the search panel then?

**Comment by lennym** [comment]
No, I'd just assume those would go to a search results page with no search term - and so all the results. No need for a separate page.

File: pages/search/views/index.jsx [hunk 1]
```diff
@@ -4,12 +4,16 @@ import { Datatable, FilterSummary, Header, Snippet } from '@asl/components';
...
 const formatters = {
-  lastName: {
-    format: (lastName, profile) => `${profile.firstName} ${lastName}`
+  establishments: {},
+  profiles: {
+    lastName: {
+      format: (lastName, profile) => `${profile.firstName} ${lastName}`
+    },
+    establishments: {
+      format: establishments => establishments.map(establishment => establishment.name).join(', ')
```
Comment (lennym) [inline pages/search/views/index.jsx]: We _might_ want to make the establishments links to the establishment page.

**Pull request merged**